test/integrationConfiguration.test.js

```diff
@@ -0,0 +1,217 @@
+        assert.strictEqual(actual.name, expected.name);
+        assert.strictEqual(actual.message, expected.message);
+
+        try {
+            await IntegrationConfiguration.getConfiguration(jsonFile, "fake-file.file");
+        } catch (e) {
+            console.log(e);
+            actual = e;
+        }
+
+        assert.strictEqual(actual.name, expected.name);
+        assert.strictEqual(actual.message, expected.message);
+
+        try {
+            process.env.ASSET_COMPUTE_PRIVATE_KEY_FILE_PATH = "fake-file.file";
+            await IntegrationConfiguration.getConfiguration(jsonFile);
+        } catch (e) {
+            console.log(e);
+            actual = e;
+        }
+
+        assert.strictEqual(actual.name, expected.name);
+        assert.strictEqual(actual.message, expected.message);
+
+
+        try {
+            process.env.ASSET_COMPUTE_INTEGRATION_FILE_PATH = "fake-file.file";
+            await IntegrationConfiguration.getConfiguration();
+        } catch (e) {
+            console.log(e);
+            actual = e;
+        }
+
+        assert.strictEqual(actual.name, expected.name);
+        assert.strictEqual(actual.message, expected.message);
+    });
+
+    it('Input file is neither JSON nor YAML', async () => {
+
+        let actual;
+        const expected = new Error("Not all integration configuration properties are present");
+
+        try {
+            await IntegrationConfiguration.getConfiguration(plainTextFile, privateKeyFile);
+        } catch (e) {
+            console.log(e);
+            actual = e;
+        }
+
+        assert.strictEqual(actual.name, expected.name);
+        assert.strictEqual(actual.message, expected.message);
+
+        try {
+            await IntegrationConfiguration.getConfiguration(plainTextFile);
+        } catch (e) {
+            console.log(e);
+            actual = e;
+        }
+
+        assert.strictEqual(actual.name, expected.name);
+        assert.strictEqual(actual.message, expected.message);
+    });
+
+    it('Input file is YAML', async () => {
+
+        let actual = await IntegrationConfiguration.getConfiguration(yamlFile);
+
+        assert.strictEqual(actual.metascopes[0], expected.metascopes[0]);
+        assert.strictEqual(actual.metascopes[1], expected.metascopes[1]);
+        assert.strictEqual(actual.metascopes[2], expected.metascopes[2]);
+
+        assert.strictEqual(actual.technicalAccount.id, expected.technicalAccount.id);
+        assert.strictEqual(actual.technicalAccount.org, expected.technicalAccount.org);
+        assert.strictEqual(actual.technicalAccount.clientId, expected.technicalAccount.clientId);
+        assert.strictEqual(actual.technicalAccount.clientSecret, expected.technicalAccount.clientSecret);
+        assert.strictEqual(actual.technicalAccount.privateKey, expected.technicalAccount.privateKey);
+
+
+        process.env.ASSET_COMPUTE_INTEGRATION_FILE_PATH = yamlFile;
+        actual = await IntegrationConfiguration.getConfiguration();
```

Comment:
Put this in another test (a separate `it(...)`. This will ensure there are no leftovers in `actual` from the test above. Leftovers render the test inconclusive: the test may pass because of old `actual` values.

The test title could be `it('Input file is YAML from env variable'`.